gulpfile.js

```diff
@@ -40,6 +41,18 @@ gulp.task('serve:dist', ['build:dist'], require('./tasks/server')(gulp, config.d
 gulp.task('serve', ['compile:src', 'less', 'fonts', 'html', 'json', 'assets'],
     require('./tasks/server')(gulp, config.srcDir, config.watchDir, true, config.projectDir, config));
 
+gulp.task('typedoc', function() {
+  return gulp
+    .src(['src/services/**/*.ts'])
```

Comment:
It gets really messy/busy and I am not sure we need those. Service API is the most important part IMHO. I am open to including if there is a decent argument for it.